package_manifest.yaml

```diff
@@ -22,14 +22,16 @@ foreman_scl_packages:
       - koji-foreman
     nightly_releaser: koji-foreman-jenkins
     repoclosure_config: repoclosure/yum_el7.conf
-    repoclosure_lookaside_repos:
-      - el7-base
-      - el7-updates
-      - el7-epel
-      - el7-extras
-      - el7-scl
-      - el7-foreman-rails-nightly
+    repoclosure_target_repos:
       - el7-foreman-nightly
+    repoclosure_lookaside_repos:
+      rhel7:
```

Comment:
So this entry has to match the end of a tag? That seems fragile.

Reply:
It's currently parsing the build target. We could try to parse the ["Build"](https://github.com/theforeman/obal/blob/master/tests/fixtures/mockbin/mockbin#L25) instead?

Reply:
I think this is fine for now. I do think we should be aiming to use tags as the connector. I think our current tito usage makes this difficult since releasers mask 1-N tags vs. iterating over a set of tags. What I am essentially saying is we should think about how we can get to a model of:

packages have tags
for each tag a repoclosure config exists
a package is built, and downloaded to a directory by tag name for each tag
repoclosure is applied to those

Reply:
Okay, so in the above example, we'd use `foreman-nightly-rhel7` as the repoclosure reposet, not `rhel7`, that part is easily done.

Now you say "repoclosure config", which we currently have split into a yum-specific config file and a set of repos in the package manifest (in different places, which does not make it easier).
I am not sure why we have multiple yum configs (the repos need to be enabled on the CLI anyways) and I think we could streamline the "enabled repos" definitions into one place (not necessarily the package manifest imho).

I think I can extend this PR to accommodate for that.